content/participate/index.html.haml

```diff
@@ -0,0 +1,119 @@
+      Participate
+
+    %p
+      As a participant, you take an active role in the community.
+
+    %h3
+      Meet
+    %p
+      Meet other Jenkins users and share your experiences.
+    %ul
+      %li
+        Organize or attend
+        %a{:href => '/projects/jam/'}
+          Jenkins Area Meetups
+      %li
+        Attend other events with a Jenkins presence, like SCALE or FOSDEM
+
+    %h3
+      Help
+    %p
+      Jenkins is a complex and extensible system, and many people are discovering it for the first time all the time. (TODO wordsmith)
+    %ul
+      %li
+        %a{:href => '/content/mailing-lists'}
+          jenkinsci-users mailing list
+      %li
+        %a{:href => '/content/chat'}
+          \#jenkins IRC channel on Freenode
+      %li
+        %a{:href => 'https://reddit.com/r/jenkinsci'}
+          \/r/jenkins on Reddit
+      %li
+        %a{:href => 'http://stackoverflow.com/tags/jenkins'}
+          Jenkins on Stack Overflow
+
+    %h3
+      Provide feedback
+    %p
+      We cannot fix what we don't know is broken. So let us know about it!
+    %ul
+      %li
+        %a{:href => 'https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue'}
+          How to report an issue in Jenkins
+      %li
+        The
+        %a{:href => '/changelog'}
+          changelog
+        has an integrated feedback mechanism. Just select
+        %em
+          Community ratings
+        and vote using the weather icons.
+      %li
+        Likewise, the
+        %a{:href => '/changelog'}
+          LTS changelog
+        provides the same functionality.
+
+  %div.col-lg-6
+    %h1
+      Contribute
+
+    %p
+      As a contributor, you actively help improve Jenkins and plugins by contributing code, documentation, translations, or tests.
+
+    %h3
+      Write code
+    %p
+      You enjoy writing code? There are numerous plugins and components for you to contribute to.
+      %a{:href => 'code'}
+        Learn more
+
+    %h3
+      Translate
+    %p
+      Jenkins is used all over the world, in numerous languages. If you speak languages other than English, you can
+      %a{:href => 'https://wiki.jenkins-ci.org/display/JENKINS/Internationalization'}
+        help make Jenkins better by contributing to core and plugin localizations.
+
+    %h3
+      Document
+    %p
+      Improve the documentation of Jenkins and plugins to make it easier for others to get started.
+
+    %ul
+      %li
+        We coordinate documenting efforts using the
+        %a{:href => 'https://groups.google.com/forum/#!forum/jenkinsci-docs'}
+          jenkinsci-docs
+        mailing list.
+      %li
+        %a{:href => 'https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-documentation'}
+          Learn how to contribute documentation to jenkins.io
+
+    %h3
+      Test
+    %p
+      The Jenkins project relies heavily on automated tests. Make sure your use cases don't break by making them into automated tests.
```

Comment:
The use of the pronoun "them" is kind of confusing to me.

Suggested rephrase, or some chopped up version of this:


> The Jenkins project relies heavily on automated tests to ensure stability and consistency of features. By contributing automated tests which match your use cases, you can help the Jenkins project prevent regressions which would affect your own Jenkins installations.